operator/crds/execution.securecodebox.io_clusterscantypes.yaml

```diff
@@ -259,8 +259,8 @@ spec:
                                         Describes node affinity scheduling
                                         rules for the pod.
                                       properties:
-                                        preferredDuringSchedulingIgnoredDuringExecution:
-                                          description:
+                                        ? preferredDuringSchedulingIgnoredDuringExecution
```

Comment:
these changes in clusterscantypes.yaml look weird to me, this doesn't seem to be in the config/crds/bases folder?

Reply:
Yeah, you're right, it's something the linter did and not the generation, MB.